src/components/form/range/_range_ticks.scss

```
@@ -22,6 +22,7 @@
 
 .euiRangeTick {
   overflow-x: hidden;
+  overflow-y: hidden;
```

I know you addressed this in your PR, description but it might be worth adding comments as well in case someone comes in later and cleans it up into just `overflow: hidden` 🤷‍♀️ 

```suggestion
  overflow-x: hidden; // Overridden if labels overflow horizontally
  overflow-y: hidden; // Should never scroll vertically
```